devicetypes/smartthings/zigbee-multi-switch-power.src/zigbee-multi-switch-power.groovy

```diff
@@ -0,0 +1,183 @@
+/*
+ *  Copyright 2019 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy
+ *  of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+
+metadata {
+	definition(name: "ZigBee Multi Switch Power", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch", mnmn: "SmartThings", vid: "generic-switch-power") {
+		capability "Actuator"
+		capability "Configuration"
+		capability "Refresh"
+		capability "Health Check"
+		capability "Switch"
+		capability "Power Meter"
+
+		command "childOn", ["string"]
+		command "childOff", ["string"]
```

Comment:
these only need to be declared if you intend to call them using tiles

since these functions are essentially only called by the DTH itself, they can be omitted

Reply:
ok, so let's leave them in case any user still uses ST classic and has that device.

Reply:
there is no reason to do that. If they're not referenced in the classic tiles they're not used.